assets/queries/k8s/kubernetes_image_digest/query.rego

```diff
@@ -0,0 +1,20 @@
+package Cx
+
+CxPolicy [ result ] {
+	document := input.document[i]
+  metadata:= document.metadata
+  spec := document.spec
+  containers := spec.containers
+  not containers[c].imagePullPolicy == "Always"
```

Comment:
I think this verification belongs to another query, here we should look into the "image" tag and check if the digest is present, the "@".